### ISSUE TITLE
vmrestore, typo in error message

### ISSUE THREAD

**Issue by Allineer** [VictoriaMetrics/VictoriaMetrics#482]
**Describe the bug**

command:
```
vmrestore -src f:///tmp/backup -storageDataPath /tmp/storage
```

error:
```
VictoriaMetrics/app/vmrestore/main.go:32	cannot parse `-src`="f:///tmp/backup": unsupported scheme "f" in `-dst`
```

I made a mistake in the `-src` parameter value, but vmrestore swears at the value of the `-dst` parameter o_O

**Version**
```
vmrestore-20200512-200628-tags-v1.35.4-0-g96e001d2
```

**Comment by valyala** [comment]
Thanks for the notification! The error message has been fixed in the commit 3412d5d138e4ab43670df50bf0f17a200429a763

**Comment by Allineer** [comment]
Thanks.
